packages/amplify_test/lib/src/integration_test_utils/auth_cognito/integration_test_auth_utils.dart

```diff
@@ -92,121 +87,107 @@ Future<void> adminCreateUser(
   List<AuthUserAttribute> attributes = const [],
 }) async {
   final config = await Amplify.asyncConfig;
-  final url = config.auth?.awsPlugin?.appSync?.default$?.apiUrl;
-  final key = config.auth?.awsPlugin?.appSync?.default$?.apiKey;
+  final api = config.api!.awsPlugin!.default$!;
   final client = GraphQLClient(
     cache: GraphQLCache(),
     link: HttpLink(
-      url!,
+      api.endpoint,
       defaultHeaders: {
-        'x-api-key': key!,
+        'x-api-key': api.apiKey!,
       },
     ),
   );
 
+  final createUserParams = <String, dynamic>{
+    'input': {
+      'autoConfirm': autoConfirm,
+      'email': attributes
+          .firstWhere(
+              (el) => el.userAttributeKey == CognitoUserAttributeKey.email,
+              orElse: () => const AuthUserAttribute(
+                  userAttributeKey: CognitoUserAttributeKey.email,
+                  value: 'example@example.com'))
+          .value,
+      'enableMFA': enableMfa,
+      'givenName': attributes
+          .firstWhere(
+              (el) => el.userAttributeKey == CognitoUserAttributeKey.givenName,
+              orElse: () => const AuthUserAttribute(
+                  userAttributeKey: CognitoUserAttributeKey.givenName,
+                  value: 'default_given_name'))
+          .value,
+      'name': attributes
+          .firstWhere(
+              (el) => el.userAttributeKey == CognitoUserAttributeKey.name,
+              orElse: () => const AuthUserAttribute(
+                  userAttributeKey: CognitoUserAttributeKey.name,
+                  value: 'default_name'))
+          .value,
+      'password': password,
+      'phoneNumber': attributes
+          .firstWhere(
+              (el) =>
+                  el.userAttributeKey == CognitoUserAttributeKey.phoneNumber,
+              orElse: () => const AuthUserAttribute(
+                  userAttributeKey: CognitoUserAttributeKey.phoneNumber,
+                  value: '+15555555'))
+          .value,
+      'username': username,
+      'verifyAttributes': verifyAttributes,
+    },
+  };
+
+  _logger.debug('Creating user "$username" with values: $createUserParams');
```

Comment:
request: Could we not log this params object here as it puts passwords in the logs? Or otherwise redact that?

Reply:
Hmm what is the risk of the password leaking?